README.md

```diff
@@ -70,7 +70,7 @@ You can always download fresh binary builds of bootable images from the ["Daily
 
 ## Installing
 
-By default, ReactOS currently can only be installed on a machine that has a FAT16 or FAT32 partition as the active (bootable) partition. 
+By default, ReactOS currently can only be installed on a machine that has a FAT16 or FAT32 partition as the active (bootable) partition.
```

Comment:
Need to mention BTRFS ?

Reply:
It's s mentioned in the next following paragraph.

Reply:
ok